src/main/java/org/sbml/spatial/segmentation/PreProcess.java

```diff
@@ -12,9 +12,9 @@
 import org.nd4j.linalg.dataset.api.preprocessor.ImagePreProcessingScaler;
 
 public class PreProcess {
-	private static final int WIDTH = 128;
-	private static final int HEIGHT = 128;
-	private static final int CHANNELS = 3;
+	public static final int WIDTH = 128;
+	public static final int HEIGHT = 128;
+	public static final int CHANNELS = 3;
```

Comment:
You only need to define it in either Inference.java or PreProcess.java. And you can reference it from anywhere

Reply:
Will make the necessary changes. Thanks.